apps/ledger-live-desktop/tests/models/SettingsPage.ts

```diff
@@ -17,6 +24,18 @@ export class SettingsPage {
     this.experimentalTab = page.locator("data-test-id=settings-experimental-tab");
     this.experimentalDevModeToggle = page.locator("data-test-id=MANAGER_DEV_MODE-button");
     this.carouselSwitchButton = page.locator("data-test-id=settings-carousel-switch-button");
+    this.counterValueSelector = page.locator(".select__value-container").first();
+    this.counterValueSearchBar = page.locator('[placeholder="Search"]');
+    this.counterValuedropdownChoiceEuro = page.locator(
+      "#react-select-2-option-15 >> text=Euro - EUR",
+    );
+    this.languageSelector = page.locator(
+      "div:nth-child(2) > .sc-dkzDqf.uXYMl > .css-198krsd-container > .select__control > .select__value-container",
```

Comment:
This is a brittle locator. If possible you should add a `data-test-id` to the component. Otherwise this is in danger of breaking with any change on the page

Reply:
emmm, i checked with @thomasrogerlux and tried serval solution to add data-test-id, not really works in this case🥶 i'll try just with text